packages/css/package.json

```diff
@@ -21,6 +23,20 @@
   },
   "private": true,
   "dependencies": {
+    "@etchteam/storybook-addon-css-variables-theme": "^1.4.0",
     "style-dictionary": "^2.10.2"
```

Comment:
This is a dev dependency as well

Any dependency that the end user doesn't have to install is a dev dependency e.g.
someone using `@compassion-gds/css` doesn't have to install `style-dictionary`